lib/backend/k8s/k8s_fv_test.go

```diff
@@ -1196,7 +1189,7 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 					CIDR:          *cidr,
 					IPIPInterface: "tunl0",
 					Masquerade:    true,
-					IPAM:          true,
```

Comment:
because you can't have both `IPAM` and `Disabled` `true` at the same time. They're `!` of each other in the conversion code.